Integrations/integration-PostgreSQL.yml

```diff
@@ -44,17 +39,22 @@ script:
           conn.close()
 
     def pgsqlConnect():
-        connectionString = "dbname='" + demisto.params()['dbname'] +"' ";
-        connectionString +="user='" + demisto.params()['user'] +"' ";
-        connectionString +="host='" + demisto.params()['host'] +"' ";
-        connectionString +="port='" + str(demisto.params()['port']) +"' ";
-        connectionString +="password='" + demisto.params()['password'] +"'";
+        params = demisto.params()
+        username = params.get('username') or params.get('credentials')['identifier']
```

Comment:
@dorsha something like this?

Reply:
Yes, just need to verify it works (and understand the using problem)